### PR TITLE
Update download_dependencies -- get L10NSharp from libpalaso

### PR DESCRIPTION
**Pull request by andrew-polk**
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/flexbridge/127)

### CODE REVIEW

File: download_dependencies_linux.sh [hunk 1]
```diff
@@ -53,14 +53,10 @@ fi
...
-f1=$(basename $1)
-f2=$(basename $2)
-cd $(dirname $2)
+f=$(basename $2)
+d=$(dirname $2)
+cd $d
 wget -q -L -N $1
-# wget has no true equivalent of curl's -o option.
-# Different versions of wget handle (or not) % escaping differently.
-# A URL query is the only reason why $f1 and $f2 should differ.
```
Comment (papeh) [inline download_dependencies_linux.sh]: looks like you're using an old version of BuildUpdate. This extra logic is needed in both scripts.

File: readme - making download_dependencies scripts.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+(Re)building buildUpdate.sh requires ruby and https://github.com/chrisvire/BuildUpdate
```
Comment (papeh) [inline readme - making download_dependencies scripts.txt]: Good idea to add these instructions. Although each generated file also has a link to chrisvire/BuildUpdate, which also has these instructions, so I feel this readme is redundant.

File: readme - making download_dependencies scripts.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+Here's the command line commands I used:
+
+cd <path to where you want to generate the update scripts>
+<your path to buildupdate.rb (part of BuildUpdate repo above)>\buildupdate.rb -t bt321 -f download_dependencies_windows.sh
+<your path to buildupdate.rb (part of BuildUpdate repo above)>\buildupdate.rb -t bt435 -f download_dependencies_linux.sh
```
Comment (papeh) [inline readme - making download_dependencies scripts.txt]: Now that the files exist, `buildupdate.rb -f <filename>` will automatically read the `bt` from the file, so there is no need to specify other options.

**Comment by andrew-polk** [comment]
Odd. I reran it just now, and it added those lines back. I don't remember updating between when I generated the PR and now, but I guess I must have. Thanks.

I'll make changes.

---

Review status: 0 of 5 files reviewed at latest revision, 2 unresolved discussions.

---

*[readme - making download_dependencies scripts.txt, line 6 at r1](https://reviewable.io:443/reviews/sillsdev/flexbridge/127#-KUN53uSW3mq0z9bjKlQ-r1-6:-KVBXeF_AdY9wrEPQSlz:b2gez6z) ([raw file](https://github.com/sillsdev/flexbridge/blob/09deaa94c255887399581e43edd1e8258c5661cf/readme - making download_dependencies scripts.txt#L6)):*
<details><summary><i>Previously, papeh wrote…</i></summary>
> Now that the files exist, `buildupdate.rb -f <filename>` will automatically read the `bt` from the file, so there is no need to specify other options.
</details>
Hm. Didn't know about this functionality. In that case, I agree this file is mostly worthless.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/sillsdev/flexbridge/127#-:-KVBYEgsqtEgiGSTbmgn:b-t1f2ae)*

**Comment by andrew-polk** [comment]
Done.

---

Review status: 0 of 4 files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/sillsdev/flexbridge/127#-:-KVBZDcCFz70sfmsDdha:b-896fix)*

**Pull request merged**